rootfs/ingest/python/qserv/contribution.py

```diff
@@ -102,11 +108,23 @@ def _build_payload(self, transaction_id: int) -> dict:
         payload = {
             "transaction_id": transaction_id,
             "table": self.table,
-            "column_separator": self.column_separator,
             "chunk": self.chunk_id,
             "overlap": self.is_overlap,
             "url": self.load_balanced_url.get(),
         }
+
+        if Contribution.fileformats is not None:
+            ff: FileFormat = Contribution.fileformats.get(self.ext)
+            if ff is not None:
```

Comment:
`ff` is declared as `FileFormat` so it's not supposed to be `None`. Maybe you need to change declaration to `Optional[FileFormat]`?

Reply:
:+1: